securedrop/source_app/decorators.py

```diff
@@ -18,7 +18,7 @@ def ignore_static(f):
     a static resource."""
     @wraps(f)
     def decorated_function(*args, **kwargs):
-        if request.path.startswith('/static'):
```

Comment:
Would moving the org-logo route behind /static make more sense here?

Reply:
We could make code changes to create a `Blueprint` whose `/static` route is set to be behind `/static/org` and whose content is exactly the single file that is uploaded. Doing this cleanly is actually just a little tricky, and our current code does not easily support this at the moment.